osc/util/packagequery.py

```diff
@@ -56,7 +56,10 @@ def read(self, all_tags = False, *extra_tags):
     @staticmethod
     def query(filename, all_tags=False, extra_rpmtags=(), extra_debtags=(), self_provides=True):
         f = open(filename, 'rb')
-        magic = f.read(7)
+        if sys.version_info >= (3, 0):
+            magic = f.read(7).decode('latin-1')
```

Comment:
Why are we forcing `latin-1` here instead of `utf-8`?

Reply:
This will be handled completely different. I do not decode here at all. I will work with the bytes